pkg/services/auth/authimpl/auth_token.go

```diff
@@ -553,22 +570,16 @@ func (s *UserAuthTokenService) GetUserRevokedTokens(ctx context.Context, userId
 }
 
 func (s *UserAuthTokenService) reportActiveTokenCount(ctx context.Context, _ *quota.ScopeParameters) (*quota.Map, error) {
-	var count int64
```

Comment:
do we know that this is more performant? if so why? 🤔

Reply:
check the sister PR to this one, it stops us from retrieving all tokens just to len them

Reply:
ah, you mean only the int and not the objects